modules/rest/src/integrationTest/java/org/apache/ignite/internal/rest/deployment/DeploymentManagementControllerTest.java

```diff
@@ -94,6 +95,7 @@ public void cleanup(TestInfo testInfo) throws Exception {
     }
 
     @Test
+    @Disabled("IGNITE-19526")
```

Comment:
IMHO, it would be better to provide a full link to the ticket "https://issues.apache.org/jira/browse/IGNITE-19526", but it's up to you.